### PR TITLE
Set agent tag

### PR DESCRIPTION
**Pull request by Acuadros95**

### CODE REVIEW

**Comment by richiprosima** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS%20Manual%20Linux/330/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS%20Manual%20Windows/350/)

**Comment by richiprosima** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS%20Manual%20Linux/332/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS%20Manual%20Windows/352/)

File: CMakeLists.txt [hunk 1]
```diff
@@ -31,7 +31,7 @@ project(microxrcedds VERSION "2.1.0")
 
 set(_client_tag develop)
 set(_client_version 2.1.0)
-set(_agent_tag develop)
+set(_agent_tag fix/delete_entities)
```
Comment (Acuadros95) [inline CMakeLists.txt:34]:
```suggestion
set(_agent_tag feature/can_flag)
```

Reuse to test https://github.com/eProsima/Micro-XRCE-DDS-Agent/pull/294

**Comment by richiprosima** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS%20Manual%20Linux/333/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS%20Manual%20Windows/353/)

**Comment by richiprosima** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS%20Manual%20Linux/335/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS%20Manual%20Windows/355/)

**Comment by richiprosima** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS%20Manual%20Linux/336/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS%20Manual%20Windows/356/)

**Comment by richiprosima** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS%20Manual%20Linux/337/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS%20Manual%20Windows/357/)

**Pull request closed without merging**